app/models/user.rb

```diff
@@ -30,7 +30,7 @@ class User < ApplicationRecord
   validates :uid, presence: true
   validates :pbs_id, presence: true, allow_blank: true
 
-  bitfield :role_flags, :role_user, :role_admin, :role_programm, :role_tn_administration
+  bitfield :role_flags, :role_user, :role_admin, :role_programm, :role_tn_administration, :role_editor
```

Comment:
one concern: what happens if we add a role? I imagined the bithash like that:

`0 0 0 0 ` (role_user, role_user, role_programm, role_programm)`, so i hope it reverts it and adds the new position to the front (i would expect a role change if you would change the order of this bitfield?

Reply:
tested the new bitfield and it works fine 👍